eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -0,0 +1,12 @@
+parameters:
+  ServiceDirectory: 'not-specified'
+
+steps:
+- pwsh: |     
+    $directory = Join-Path "$(Build.SourcesDirectory)" "sdk"
+    $directory = Join-Path $directory "${{parameters.ServiceDirectory}}"
+    Write-Host "directory = $directory"
+    foreach ($md in $(dir $directory -r -i *.md)){
+      $(Build.SourcesDirectory)/eng/common/scripts/Verify-Links.ps1 $md
```

Comment:
This will cause failure on the first md that has bad links. If we want this to report all the bad links the you will need to handle error codes and change error handling.